data/SinghRamesh_2023/metadata.yml

```diff
@@ -0,0 +1,459 @@
+  unit_in: '%'
+  trait_name: leaf_C_per_dry_mass
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: For both field-collected, and glasshouse grown leaves, leaf area was calculated
+    from scanned fresh leaves (3 to 5 leaves per individual) using Image-J software,
+    which in conjunction with oven-dried leaf mass was used to calculate LMA (g m2).
+    Further, ovendried leaves were ground to a fine powder using a Bench Top Ring
+    Mill. Foliar d13C and d15N isotope ratios were determined using a Costech Elemental
+    Analyzer, fitted with a zero-blank autosampler and coupled via a ConFloIV interface
+    to a Thermo Finnigan Delta-V PLUS isotope ratio mass spectrometer (Bremen, Germany).
+    Stable isotope results are reported as per mil (h) deviations from the VPDB and
+    AIR reference standard scales for d13C and d15N, respectively.
+- var_in: '%N'
+  unit_in: '%'
+  trait_name: leaf_N_per_dry_mass
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: For both field-collected, and glasshouse grown leaves, leaf area was calculated
+    from scanned fresh leaves (3 to 5 leaves per individual) using Image-J software,
+    which in conjunction with oven-dried leaf mass was used to calculate LMA (g m2).
+    Further, ovendried leaves were ground to a fine powder using a Bench Top Ring
+    Mill. Foliar d13C and d15N isotope ratios were determined using a Costech Elemental
+    Analyzer, fitted with a zero-blank autosampler and coupled via a ConFloIV interface
+    to a Thermo Finnigan Delta-V PLUS isotope ratio mass spectrometer (Bremen, Germany).
+    Stable isotope results are reported as per mil (h) deviations from the VPDB and
+    AIR reference standard scales for d13C and d15N, respectively.
+- var_in: Leaf_N_Area_recalculated_g/m2
+  unit_in: g/m2
+  trait_name: leaf_N_per_area
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: For both field-collected, and glasshouse grown leaves, leaf area was calculated
+    from scanned fresh leaves (3 to 5 leaves per individual) using Image-J software,
+    which in conjunction with oven-dried leaf mass was used to calculate LMA (g m2).
+    Further, ovendried leaves were ground to a fine powder using a Bench Top Ring
+    Mill. Foliar d13C and d15N isotope ratios were determined using a Costech Elemental
+    Analyzer, fitted with a zero-blank autosampler and coupled via a ConFloIV interface
+    to a Thermo Finnigan Delta-V PLUS isotope ratio mass spectrometer (Bremen, Germany).
+    Stable isotope results are reported as per mil (h) deviations from the VPDB and
+    AIR reference standard scales for d13C and d15N, respectively.
+- var_in: Leaf_P_Area_recalculated_g/m2
+  unit_in: g/m2
+  trait_name: leaf_P_per_area
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: unknown
```

Comment:
How come the methods for `leaf_P_per_area`, `stem_delta13C`  and `leaf_P_per_dry_mass` are unknown when `stem_delta15N` and `leaf_N_per_are` have methods? Or is it just the phosphorus measurements that are unknown?

Reply:
just the `leaf_P` traits should have been unknown